indra/statements/io.py

```diff
@@ -235,32 +246,38 @@ def pretty_print_stmts(stmt_list: List[Statement],
     ev_limit :
         The maximum number of Evidence to print for each Statement.
         (Default is 5)
+    width :
+        Manually set the width of the table. Default is None, in which case
+        the function will try to match the current terminal width, and if that
+        is not possible it will use `pretty_print_default_width`, which can be
+        adjusted using the `set_pretty_print_default_width` function.
```

Comment:
Also for sphinx linking

```suggestion
        adjusted using the :func:`set_pretty_print_default_width` function.
```